.prettierrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+	quoteProps: 'consistent',
```

Comment:
I’m voting in favor of removing this since this will cause bigger diffs for no benefit that’s known to me. Imagine someone adds a `background-color` to an already existing style object. Now, the entire object gets re-formatted.

Reply:
i aligned with yoda. Not attached to this at all, but IMO we should align all repos to use similar settings (i don't care which) @mvangent?

Reply:
I agree that repos should be aligned, so also voting in favor of removing it in Yoda.

Reply:
@FlorianWendelborn in that case which option would we consider: 'as needed', maybe?

Reply:
well, feel free to push to this PR with whatever you guys decide - but we will then need to reformat yoda. Might be a good time to do a full linting as well, and get the big diff over with.

Reply:
I agree as-needed (the default) is preferred over consistent.

Reply:
as-needed it is then